### ISSUE TITLE
phi-1 == phi-1.5 ?

### ISSUE THREAD

**Issue by zhoumengbo** [kyegomez/phi-1#3]
Has anyone run this project? Dear author, do you think the code for phi-1 and phi-1.5 is completely identical, with the only difference being the training datasets? I would like to know if the project code can support me in training phi-1.5 from scratch.

**Comment by kyegomez** [comment]
@zhoumengbo Yes the model architecture is the same the difference is dataset.

**Comment by zhoumengbo** [comment]
> @zhoumengbo Yes the model architecture is the same the difference is dataset.

I really like your work. Did you use flashAttention 2 in your implementation? How are the results of your pre-training, and can it reach the level mentioned in the paper?
